src/main/java/blackjack/Application.java

```diff
@@ -0,0 +1,47 @@
+package blackjack;
+
+import blackjack.controller.BlackJackController;
+import blackjack.dto.ParticipantDto;
+import blackjack.view.input.DrawCommand;
+import blackjack.view.input.InputView;
+import blackjack.view.output.OutputView;
+import java.util.List;
+
+public class Application {
```

Comment:
메인 클래스가 너무 복잡합니다.
아래 코드와 같이 단순하게 만들어봅시다.

```suggestion
public class Application {
    public static void main(final String[] args) {
        var blackJackGame = new BlackJackGame();
        blackJackGame.run();
    }
}
```